helm/charts/infra/README.md

```diff
@@ -0,0 +1,70 @@
+# Infra Chart
+
+A Helm chart for Infra, an infrastructure access management tool for Kubernetes.
+
+The default deployment installs `infra-server` and accompanying services including a frontend service and a PostgreSQL database. The full list of configuration values can be found with `helm show values infrahq/infra`.
+
+## Upgrade
+
+The upgrade is a standard Helm upgrade. Run the commands below to update the Helm repository and perform the upgrade. Ensure the correct values are specified to avoid any unnecessary changes to the deployment. As in the previous step, these commands assume the Helm release is called `infra`, update as needed.
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+HELM_ARGS=.. # additional arguments to helm, e.g. --set or --values
+
+### helm repo update infrahq
+helm -n $RELEASE_NAMESPACE upgrade $RELEASE_NAME infrahq/infra $HELM_ARGS
+```
+
+Once upgraded, verify the deployment with `infra login` or the Infra UI. Once verified, the server persistent volume claim should be disabled, `server.persistence.enabled=false`, to fully remove the SQLite database and old database encryption key.
+
+## Rollback
+
+If the upgrade is unsuccessful, the deployment can be rolled back using Helm. The upgrade can be attempted again or you can contact us through GitHub for additional assistance.
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+
+helm -n $RELEASE_NAMESPACE rollback $RELEASE_NAME
+```
+
+### 0.14.4
+
+Infra chart version 0.20.0 introduces two backwards incompatible changes. The first is the migration to PostgreSQL as the default backend database from SQLite. The second is the migration of the database encryption key from within the server PVC to a Kubernetes secrets. This guide will detail the required steps to successfully migrate an Infra deploy to 0.20.0.
```

Comment:
Maybe:

```Infra chart version 0.20.0 and greater introduces two backward incompatible changes from 0.14.4 and earlier.```
... and:
```This guide will show you the required steps to upgrade from earlier versions.```

I'm thinking we don't want to be specific about 0.20.0 because it will be out of date pretty quickly.

Reply:
I think it's important to highlight that 0.20.0 introduces these changes. If we remove the migration at a later stage, these instructions will no longer be applicable so if a user upgrades, they will need to make a pit stop at 0.20.0. This section is intended to be upgrading from 0.14.4 but maybe the heading should be to `0.20.0`

Reply:
I agree. We should definitely call out that 0.20.0 introduces the changes. My concern was we're going to move past 0.20.0 quickly with different releases and admins will not realize the changes are applicable to them if they skip 0.20.0.

I think you're right that the section should be called something like `### Version 0.20.0`, or maybe `### Version 0.20.0 and greater`. 

I'm not as worried about pulling the section out in the future. If we remove it in ~4-6 months I doubt there will still be anyone using the older versions.